examples/llm/README.md

```diff
@@ -78,12 +78,15 @@ To make yourself a copy of C4, use `convert_c4.py` like so:
 # Download the 'train_small' and 'val' splits and convert to StreamingDataset format
 # This will take 20-60 seconds depending on your Internet bandwidth
 # You should see two folders: `./my-copy-c4/train_small` and `./my-copy-c4/val` that are each ~0.5GB
-python ../common/convert_c4.py --out_root ./my-copy-c4 --splits train_small val
+# Note: We are using the `--concat_tokens` option to pre tokenize our samples to be of the max sequence length without padding
+# `--concat_text` is also an option if you don't want to pre tokenize, but may result in some padding or truncated text in your samples,
+# which affects the MFU calculation
+python ../common/convert_c4.py --out_root ./my-copy-c4 --splits train_small val --concat_tokens 2048 --tokenizer gpt2 --bos_text '<|endoftext|>'
```

Comment:
I don't think we want `bos_text` here, we want `eos_text`.

Reply:
oh yeah, woops

Reply:
Honestly, if I was going to do one, I'd use it as BOS. Since we are just picking one, it won't be any different at the boundary between two sequences. The difference is only for the first segment in the sequence. And at the beginning of the sequence, I think it will just have to use absolute unigram probabilities for the first token, rather than the distribution given that it is the first token in the sequence (correct me if I'm wrong there). Here is an n=1 experiment in favor of this POV https://github.com/huggingface/transformers/issues/3311

Reply:
@abhi-mosaic thoughts on me reverting to bos?

Reply:
I am unopinionated, although gpt paper does call it an EOS, not a BOS